frontend/src/metabase/admin/settings/notifications/NotificationSettings.tsx

```diff
@@ -0,0 +1,134 @@
+import { useState } from "react";
+import { Link } from "react-router";
+import { t } from "ttag";
+
+import { useListChannelsQuery } from "metabase/api/channel";
+import {
+  Box,
+  Button,
+  Flex,
+  Icon,
+  type IconName,
+  Paper,
+  Stack,
+  Text,
+  Title,
+} from "metabase/ui";
+import type { NotificationChannel } from "metabase-types/api";
+
+import { CreateWebhookModal } from "./CreateWebhookModal";
+import { EditWebhookModal } from "./EditWebhookModal";
+
+type NotificationModals = null | "create" | "edit";
+
+export const NotificationSettings = () => {
+  const [webhookModal, setWebhookModal] = useState<NotificationModals>(null);
+  const [currentChannel, setCurrentChannel] = useState<NotificationChannel>();
+
+  const { data: channels } = useListChannelsQuery();
+
+  return (
+    <>
+      <Box w="47rem">
+        <Title mb="1.5rem">{t`Slack`}</Title>
+        <Link to="/admin/settings/notifications/slack">
+          <Paper shadow="0" withBorder p="lg" w="47rem" mb="2.5rem">
+            <Flex gap="0.5rem" align="center" mb="0.5rem">
+              <Icon name="slack_colorized" />
+              <Title order={2}>{t`Connect to slack`}</Title>
+            </Flex>
+            <Text>
+              {t`If your team uses slack, you can send dashboard subscriptions and
+            alerts there`}
+            </Text>
+          </Paper>
+        </Link>
+
+        <Flex justify="space-between" align="center" mb="1.5rem">
+          <Title>{t`Alerts webhook`}</Title>{" "}
+          {channels?.length !== 0 && (
+            <Button
+              variant="subtle"
+              compact
+              leftIcon={<Icon name="add" />}
+              onClick={() => setWebhookModal("create")}
+            >{t`Add another`}</Button>
```

Comment:
```suggestion
            >{c("Short for 'Add another webhook'").t`Add another`}</Button>
```

The translation of 'another' may vary since the noun is 'webhook'